vm/actor/src/builtin/miner/policy.rs

```diff
@@ -152,11 +154,17 @@ pub fn reward_for_consensus_slash_report(
     let slasher_share_denominator = consensus_fault_reporter_share_growth_rate
         .denominator
         .pow(&elapsed);
-    let num =
-        (slasher_share_numerator * consensus_fault_reporter_initial_share.numerator) * &collateral;
-    let denom = slasher_share_denominator * consensus_fault_reporter_initial_share.denominator;
+    let num = BigInt::from_biguint(
```

Comment:
change the values above to big int so you don't have to convert from biguint on these